docpage-CSS/typography.css

```diff
@@ -0,0 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Roboto+Slab&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&display=swap');
+
+
+
+body{
+font-family: 'Bree Serif', serif;
+font-family: 'Montserrat', sans-serif;
+font-family: 'Roboto Slab', serif;
+font-family: 'Recursive', sans-serif;
+}
```

Comment:
I am not getting why u gave all this font-family 
plus you can make variables for font-family as you did for colors that will be handy

Reply:
Okay I will look at this.Thankyou for your valuable feedback.